jvm/src/main/scala/eval/ParallelInterpreter.scala

```diff
@@ -0,0 +1,62 @@
+package com.azavea.maml.eval
+
+import com.azavea.maml.ast._
+import com.azavea.maml.error._
+import com.azavea.maml.eval.directive._
+
+import cats._
+import cats.implicits._
+import cats.data.Validated._
+import cats.data.{NonEmptyList => NEL, _}
+import cats.effect.{Concurrent, Fiber}
+import cats.syntax.semigroup._
+
+import scala.reflect.ClassTag
+
+class ParallelInterpreter[F[_]](directives: List[Directive])(
```

Comment:
This doesn't extent `Interpreter` because `Interpreter` has some strong opinions about whether there are effects anywhere. I thought I'd be able to get around this by calling an `unsafeRunSync` at the end, but that's only defined on the `SyncIO` and `IO` classes, which meant no more generality over `F[_]`

Reply:
Just kidding it extends `Interpreter[F]` now